sickbeard/providers/generic.py

```diff
@@ -274,13 +274,13 @@ def findSearchResults(self, show, episodes, search_mode, manualSearch=False):
                 continue
 
             # skip if season already searched
-            if len(episodes) > 1 and searched_scene_season == epObj.scene_season:
+            if len(episodes) > 1 and searched_scene_season == epObj.scene_season and not(episodes and sickbeard.TORRENT_METHOD == 'transmission'):
```

Comment:
@abeloin i've used this method (sickbeard.TORRENT_METHOD == 'transmission') to know which client is used that is hard coded. There is other solution through sinlge_episode_enable procedure in clients.
the second way i think is better in order to have other clients to support it but we have to instantiate the client in orther to have this information. The first solution probably is better for performance. let me know.